README.md

```diff
@@ -39,6 +39,12 @@ The site should now be live at `http://localhost:4000/capital-framework/`.
 $ grunt vendor
 ```
 
+**Deny/Allow connections.** If you are an employee of the CFPB, you may encounter the following message:
```

Comment:
It happened on Mountain Lion, too. I'm guessing that on your personal machine or whatever, you saw it once, hit Allow or Deny, and it remembered your choice, because you had root privileges. Our users can't set the option permanently on these machines.

So, not a high likelihood that regular non-CFPB users will see it, but you never know.

Thanks!